__test__/game.test.js

```diff
@@ -2,11 +2,10 @@ const Game = require('../src/game').default
 const fs = require('fs')
 
 describe('App', () => {
-  it('Contains the compiled JavaScript', async (done) => {
```

Comment:
"Test functions cannot both take a 'done' callback and return something. Either use a 'done' callback, or return a promise."